app/src/hooks/connectedWeb3.tsx

```diff
@@ -1,89 +0,0 @@
-import { providers } from 'ethers'
-import React, { useEffect, useState } from 'react'
-import { useWeb3Context } from 'web3-react'
-
-import connectors from '../util/connectors'
-import { getLogger } from '../util/logger'
-
-const logger = getLogger('Hooks::ConnectedWeb3')
-
-export interface ConnectedWeb3Context {
-  account: Maybe<string>
-  library: providers.Web3Provider
-  networkId: number
-  rawWeb3Context: any
-}
-
-const ConnectedWeb3Context = React.createContext<Maybe<ConnectedWeb3Context>>(null)
-
-/**
- * This hook can only be used by components under the `ConnectedWeb3` component. Otherwise it will throw.
- */
-export const useConnectedWeb3Context = () => {
-  const context = React.useContext(ConnectedWeb3Context)
-
-  if (!context) {
-    throw new Error('Component rendered outside the provider tree')
-  }
-
-  return context
-}
-
-/**
- * Component used to render components that depend on Web3 being available. These components can then
- * `useConnectedWeb3Context` safely to get web3 stuff without having to null check it.
- */
-export const ConnectedWeb3: React.FC = props => {
-  const [networkId, setNetworkId] = useState<number | null>(null)
-  const context = useWeb3Context()
-  const { account, active, error, library } = context
-
-  useEffect(() => {
-    let isSubscribed = true
-    const connector = localStorage.getItem('CONNECTOR')
```

Comment:
I noticed that one thing that we lose with the deletion of this file is auto login / auto connector detection on refresh.

I'm also using this file to [auto login safes](https://github.com/protofire/omen-exchange/pull/1317/commits/f411ddd13626eac89074afee69ecd37053b73c58#diff-1c8e03f884839b8a2150732e6e176df8c8fae34038fef297837bae585c6bd95cR53) in https://github.com/protofire/omen-exchange/pull/1317

Is there anyway we can do the upgrade without deleting this?